app/views/questions/show.html.erb

```diff
@@ -0,0 +1,82 @@
+<div id="question-show">
+	<h3><%= @question.title %></h3>
+
+	<article>
+	<p><%= @question.content %></p>
+	<div id="score">
+		<div id="question-voting">
+			<% if current_user.hasnt_voted?("Question", @question.id) %>
```

Comment:
Think about pulling your three types of vote form out into a set of partials. Once you've done that, look at the partials and see if you can reduce their number (maybe by passing in the param to form_for as an array in the locals first.)